### ISSUE TITLE
Gutenberg: investigate which TinyMCE plugins we need to import

### ISSUE THREAD

**Issue by vindl** [Automattic/wp-calypso#27883]
During the investigation of [List block issue](https://github.com/Automattic/wp-calypso/issues/27637) we discovered that the underlying cause was missing TinyMCE plugin. We are suspecting that we might be missing other plugins to, so we need to perform an audit in determine if there are others that should be imported in the similar fashion.

Here is the above fix for reference: https://github.com/Automattic/wp-calypso/pull/27877

And here is the list of plugins that are currently enqueued in core: https://github.com/WordPress/gutenberg/blob/de2fab7b8d66eea6c1aeb4a51308d47225fc5df8/lib/client-assets.php#L494

As a follow up, we'll also need to start an issue in core to make sure that TinyMCE plugins can be loaded from npm modules.

**Comment by kwight** [comment]
@vindl I'm not sure how to go about doing this. Looking at the [list of plugins in core](https://github.com/WordPress/gutenberg/blob/master/lib/client-assets.php#L584), `charmap` and `fullscreen` are missing from the [TinyMCE component](https://github.com/Automattic/wp-calypso/blob/master/client/components/tinymce/index.jsx#L15), but I don't think that's related here?

The fix you pointed to involves importing a plugin that handles broken functionality; are you suggesting we should be somehow importing all of the listed core plugins into the new `init.js` from [yesterday](https://github.com/Automattic/wp-calypso/pull/28302)?.. I know not what I speak of 🙃

**Comment by vindl** [comment]
> The fix you pointed to involves importing a plugin that handles broken functionality; are you suggesting we should be somehow importing all of the listed core plugins into the new init.js from yesterday?.. I know not what I speak of 🙃

I don't think we should import all of them. At the time of the creation of this issue we suspected that there might be more broken functionality in editor integration due to missing TinyMCE plugins. The only idea I had for checking this was to audit why some of the plugins were added in core, and then test equivalent functionality in Calypso.

However, now that we are importing TinyMCE component via Classic block, it's increasingly less likely that some of the required plugins are not loaded. Taking that into account, I suggest that we close this issue and address bugs if they pop up in testing on case by case basis.

For reference, this was also briefly discussed in p7jreA-1Se-p2. The core is moving toward removing all of the TinyMCE plugins, so this will hopefully become a non-issue.
